src/controllers/dynakube/dynakube_phase.go

```diff
@@ -0,0 +1,86 @@
+package dynakube
+
+import (
+	"context"
+
+	dynatracev1beta1 "github.com/Dynatrace/dynatrace-operator/src/api/v1beta1"
+	"github.com/Dynatrace/dynatrace-operator/src/controllers/activegate/capability"
+	appsv1 "k8s.io/api/apps/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+func (controller *DynakubeController) determineDynaKubePhase(instance *dynatracev1beta1.DynaKube) bool {
```

Comment:
```suggestion
func (controller *DynakubeController) determineDynaKubePhase(dynakube *dynatracev1beta1.DynaKube) bool {
```
since we are dealing with a `dynakube`, we should also name the variable like that. `instance` could be any instance